GSA/digitalgov.gov
# Feature: Update logic to remove ?dg

## Summary

Removing the set up to add `?dg` from urls to help clean up tracking within Google Analytics 



### Solution

Removed all mentions of `?dg`, this includes notes that were adding warning the user of the addition -> commit https://github.com/GSA/digitalgov.gov/pull/7490/commits/07846b6c1e54fc9a7667ddc67c4fec0473357f07

Removed any urls that were hardcoded in with `?dg` -> commit https://github.com/GSA/digitalgov.gov/pull/7490/commits/71cf3d2376b49b3950cf805070725938b58cea18

Code updates:
Commit -> https://github.com/GSA/digitalgov.gov/pull/7490/commits/e564d29a6d49a0baf68db297d74cb8a627975432
- Removed the print statement to add `?dg` to the url 
- Removed if statements that were also adding `?dg`

Commit -> https://github.com/GSA/digitalgov.gov/pull/7490/commits/9ab0c1b6c587e8cfc49169cae4299f6ab6085099
- Removed script that added `?dg` with location.href

Commit -> https://github.com/GSA/digitalgov.gov/pull/7490/commits/2b316b0341e55da4e1c06ceb5b47f29333f4947a
- Added redirect for anyone that might have saved urls that were hard coded

### Note
By using `?` in the url it sets up the use of a query selector. The use of `?dg` was intended to help with track for other users and the traffic that went to the site 

### How To Test

1. Pull down branch and search for `?dg` within the project and ensure there are no mentions of the string

---

### Dev Checklist
